cicd/gitops/health-information-for-resources-deployment.adoc

```diff
@@ -6,6 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The environment details page displays the health status of the application resources, such as routes, synchronization status, deployment configuration and deployment history.
+The {gitops-title} *Environments* page in the *Developer* perspective of the {product-title} web console shows a list of the successful deployments of the application environments, along with links to the revision for each deployment.
+
+The *Application environments* page in the *Developer* perspective of the {product-title} web console displays the health status of the application resources, such as routes, synchronization status, deployment configuration, and deployment history.
+
+The *Application environments* pages in the *Developer* perspective of the {product-title} web console are decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. You do not have to use `kam` to generate the Application Environment manifests in order for the environments to show up in the *Developer* perspective of the {product-title} web console. You can use your own manifests, but the environments must however still be represented by namespaces. In addition, specific labels and annotations are still needed.
```

Comment:
I would suggest to use KAM instead of kam

Reply:
Hi, @varshab1210: Thanks for the suggestion. PTAL at [this](https://redhat-internal.slack.com/archives/C01RQH8KQ87/p1677854833011389?thread_ts=1677589630.738149&cid=C01RQH8KQ87) thread.

Just like how the `tkn` CLI is addressed in our OCP docs for Pipelines, we will be following the same principle. Instances of:

- kam will change to: the `kam` CLI
- Red Hat OpenShift GitOps Application Manager (kam) will change to: Red Hat OpenShift GitOps Application Manager CLI (`kam`)

Reply:
@varshab1210 and @ciiay: The wording for the `kam` CLI  - https://issues.redhat.com/browse/RHDEVDOCS-5030, is handled as part of the [GitOps RN v1.8 PR](https://github.com/openshift/openshift-docs/pull/55921).